test/unit/behaviors.spec.js

```diff
@@ -315,7 +315,7 @@ describe('Behaviors', function() {
       });
 
       it('should call onShow when already shown and reset', function() {
-        this.collection.reset([{id:1}, {id: 2}]);
+        this.collection.reset([{id: 1}, {id: 2}]);
```

Comment:
The newest version of jscs complained about the missing space here, and it seems most other singleline objects have the space so I added it.